<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWriteMovieFrames</class>
 <widget class="QDialog" name="DialogWriteMovieFrames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write Movie Frames</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Move this window out of the main viewport before starting to write.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="labelStartNumber">
       <property name="text">
        <string>Start number</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBoxStep">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditFilenamePrefix">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>frame</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBoxEnd">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxFlyThrough">
       <item>
        <property name="text">
         <string>Slice</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Frame</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Angle</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filename prefix</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelEndNumber">
       <property name="text">
        <string>End number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxExtension">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpeg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tiff</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output location</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxStep">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.250000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBoxStart">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filename extentsion</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelSliceStep">
       <property name="text">
        <string>Step size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelFlyThrough">
       <property name="text">
        <string>Fly through</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditOutputLocation">
       <property name="minimumSize">
        <size>
         <width>215</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelSliceStepDouble">
       <property name="text">
        <string>Step size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButtonOpen">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelRotateDirection">
       <property name="text">
        <string>Direction of rotation</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxRotateDirection">
       <item>
        <property name="text">
         <string>Horizontal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Elevation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Roll</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QLabel" name="labelStepSizeNote">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>(Negative value for opposite direction)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxShowAnnotations">
     <property name="text">
      <string>Show annotations</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>NOTE: Angle is for spinning in 3D view only. If the main viewport is 2D slice view, it will be ignored. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAbort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWrite">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogWriteMovieFrames</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpen</sender>
   <signal>clicked()</signal>
   <receiver>DialogWriteMovieFrames</receiver>
   <slot>OnOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAbort</sender>
   <signal>clicked()</signal>
   <receiver>DialogWriteMovieFrames</receiver>
   <slot>OnAbort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonWrite</sender>
   <signal>clicked()</signal>
   <receiver>DialogWriteMovieFrames</receiver>
   <slot>OnWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogWriteMovieFrames</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFlyThrough</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogWriteMovieFrames</receiver>
   <slot>OnComboBoxFlyThrough(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOpen()</slot>
  <slot>OnWrite()</slot>
  <slot>OnAbort()</slot>
  <slot>OnComboBoxFlyThrough(int)</slot>
 </slots>
</ui>
